### ISSUE TITLE
Insufficient colour contrast for messages

### ISSUE THREAD

**Issue by chmst** [joomla-projects/gsoc25_workflow#14]
### Steps to reproduce the issue
I suggest to make it simple and use the normal text colour for every test. 
or: 
remove the message completely. 
or: 
add it into the definition list.

what is your opinion? If you are user, what can you do with the message?,

<img width="573" height="149" alt="Image" src="https://github.com/user-attachments/assets/4336d9f6-c293-4e95-8135-f3ee68255249" />

**Comment by Dileepadari** [comment]
I think having it lets user know their status, we can have a time limit say 3 secs after saving, just remove the message and when user changes anything then have the message again. What do you think?

**Comment by chmst** [comment]
Yes, we could do this. 
But he main concern is the colour contrast which gives an accessibility error, you can see it yourself in WAVE. This needs to be fixed.
